src/lib/y2users/config.rb

```diff
@@ -18,9 +18,22 @@
 # find current contact information at www.suse.com.
 
 module Y2Users
-  # Holds references to elements of user configuration like users or groups.
-  # Class itself holds references to different configuration instances.
-  # TODO: write example
+  # Class to represent a configuration of users and groups
+  #
+  # @example
+  #   user = User.new("john")
+  #   group = Group.new("users")
+  #
+  #   config1 = Config.new("config1")
+  #   config1.users #=> []
+  #   config1.attach(user, group)
```

Comment:
I am not sure if this does not create impression that you need to attach user and group together

Reply:
Ok, I will modify the example.